.github/workflows/validation.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint and Tests 
+
+on: push 
```

Comment:
It will not work on the pull requests.
Can you please copy relevant parts of https://github.com/graphql/graphql-js/blob/master/.github/workflows/ci.yml
It will simplify review and unify setups between projects.

Reply:
I will make the change you are asking. but it is not necessary, It will run every time we push to the branch. 

as an example [this is the same PR to my fork](https://github.com/gagoar/swapi-graphql/pull/2). you will see is running, 

Is not running here,  because the given permissions on the repo.

Reply:
the issue here. is if I add both, GitHub have some issues and it creates duplicated runs. (https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012)

Reply:
![image](https://user-images.githubusercontent.com/226559/100256913-30c74700-2efa-11eb-952e-1a86c4f0b73b.png)
We end up with double runs now in my fork, which is not desired.